Source/WebKit/UIProcess/CoordinatedGraphics/DrawingAreaProxyCoordinatedGraphics.cpp

```diff
@@ -90,22 +90,21 @@ bool DrawingAreaProxyCoordinatedGraphics::forceUpdateIfNeeded()
 {
     ASSERT(!isInAcceleratedCompositingMode());
 
-    auto webPageProxy = protectedWebPageProxy();
-    if (!webPageProxy->hasRunningProcess())
+    if (!webProcessProxy->hasConnection())
         return false;
 
-    if (webPageProxy->process().state() == WebProcessProxy::State::Launching)
+    if (webProcessProxy->state() == WebProcessProxy::State::Launching)
         return false;
 
     if (m_isWaitingForDidUpdateGeometry)
         return false;
 
-    if (!webPageProxy->isViewVisible())
+    if (!protectedWebPageProxy()->isViewVisible())
         return false;
 
     SetForScope inForceUpdate(m_inForceUpdate, true);
-    webPageProxy->send(Messages::DrawingArea::ForceUpdate(), m_identifier);
-    webPageProxy->process().connection()->waitForAndDispatchImmediately<Messages::DrawingAreaProxy::Update>(m_identifier, Seconds::fromMilliseconds(500));
+    send(Messages::DrawingArea::ForceUpdate());
+    webProcessProxy->connection()->waitForAndDispatchImmediately<Messages::DrawingAreaProxy::Update>(m_identifier, Seconds::fromMilliseconds(500));
```

Comment:
Since you're modifying this line, `500_ms` would look nicer than `Seconds::fromMilliseconds(500)`